### PR TITLE
remove fanstatic asset management.

### PR DESCRIPTION
**Pull request by RemiDesgrange**
Let css files be served via the public repo and called by a simple link
html tag. This way we are sure that css are called the right way. We
concat, but it works :-)

### CODE REVIEW

File: ckanext/pigma_theme/plugin.py [hunk 1]
```diff
@@ -23,7 +23,6 @@ class Pigma_ThemePlugin(plugins.SingletonPlugin):
     def update_config(self, config_):
         toolkit.add_template_directory(config_, 'templates')
         toolkit.add_public_directory(config_, 'public')
-        toolkit.add_resource('fanstatic', 'pigma_theme')
```
Comment (tonio) [inline ckanext/pigma_theme/plugin.py]: I think that will break link to images in that folder.

File: ckanext/pigma_theme/templates/page.html [hunk 2]
```diff
@@ -128,8 +128,8 @@ <h1>My page content</h1>
 {%- endblock -%}
 
 {%- block scripts %}
-    {% resource 'base/main' %}
-    {% resource 'base/ckan' %}
+   {% resource 'base/main' %}
+   {% resource 'base/ckan' %}
```
Comment (tonio) [inline ckanext/pigma_theme/templates/page.html]: nope (-:

File: ckanext/pigma_theme/templates/base.html [hunk 3]
```diff
@@ -2,8 +2,8 @@
...
 {% block styles %}
   {{ super() }}
-  {% resource 'pigma_theme/font.css' %}
-  {% resource 'pigma_theme/pigma_main.css' %}
-  {% resource 'pigma_theme/pigma_styles.css' %}
-  {% resource 'pigma_theme/pigma_theme.css' %}
+  <link rel="stylesheet" type="text/css" href="/css/font.css">
+  <link rel="stylesheet" type="text/css" href="/css/pigma_main.css">
+  <link rel="stylesheet" type="text/css" href="/css/pigma_styles.css">
+  <link rel="stylesheet" type="text/css" href="/css/pigma_theme.css">
```
Comment (tonio) [inline ckanext/pigma_theme/templates/base.html]: `type` attribute is not necessary anymore.

File: ckanext/pigma_theme/plugin.py (hunk 1 above)
Comment (RemiDesgrange) [reply ckanext/pigma_theme/plugin.py]: I'll check this out. but it didn't break anything on my setup so giving it a closer look

File: ckanext/pigma_theme/templates/page.html (hunk 2 above)
Comment (RemiDesgrange) [reply ckanext/pigma_theme/templates/page.html]: WDYM ? formating ?

Comment (tonio) [reply ckanext/pigma_theme/templates/page.html]: yep

File: ckanext/pigma_theme/plugin.py (hunk 1 above)
Comment (tonio) [reply ckanext/pigma_theme/plugin.py]: I guess you don't have locally the groups using them

**Review by tonio — APPROVED** [review]
LGTM. Maybe build & deploy to check on production instance before merging.

**Pull request merged**